src/components/card/card.njk

```diff
@@ -1,5 +1,5 @@
-{% from "badge/large/macro.njk" import nhsappBadgeLarge %}
-{% from "icon/macro.njk" import nhsappIcon %}
+{% from "../badge/large/macro.njk" import nhsappBadgeLarge %}
+{% from "../../styles/icon/macro.njk" import nhsappIcon %}
```

Comment:
I assume these changes work for both the documentation here and then the prototype repo?

Reply:
Never assume @edwardscull 😜

It works for me locally:
<img width="771" alt="Screenshot 2024-08-23 at 10 18 20" src="https://github.com/user-attachments/assets/c71a4093-23ed-4f8a-a046-7ada3ca51c0f">

You might want to test it too, just to be sure.